.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\diana\\AppData\\Local\\Programs\\Python\\Python38\\python.exe"
+}
```

Comment:
Remove `.vscode` from git (you can add it to `.gitignore` or, if you don't want that pushed to the remote repo, you can put it in `.git/info/exclude` instead, which is kind of a gitignore but only for you)